### PR TITLE
Mark's story time submission about the monkey!

### PR DESCRIPTION
**Pull request by markhirsch91**
Here's how the story concludes!

### CODE REVIEW

**Review by zachherington — COMMENTED** [review]
nice!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # project1-repo
 
+
 Chapter 1:
 
 One a cold winter night, Hank the monkey was really bored. He decided to make a homemade flamethrower. 
-The END. 
\ No newline at end of file
+The END.
```
Comment (zachherington) [inline README.md]: hey this is not the end.

**Review by zachherington — CHANGES_REQUESTED** [review]

**Review by Tomorrowrose — APPROVED** [review]
looks good!

**Review by zachherington — APPROVED** [review]
nice!

**Pull request merged**